param/__init__.py

```diff
@@ -1376,6 +1383,15 @@ def __init__(self, default=[], class_=None, item_type=None,
                            **params)
         self._validate(default)
 
+    def _validate(self, val):
+        """
+        Checks that the value is numeric and that it is within the hard
+        bounds; if not, an exception is raised.
+        """
```

Comment:
Oops; left a useless docstring in from where I copied this.  I'll delete in a later PR.